detections/application/splunk_command_and_scripting_interpreter_risky_commands.yml

```diff
@@ -0,0 +1,66 @@
+name: Splunk Command and Scripting Interpreter Risky Commands
+id: 1cf58ae1-9177-40b8-a26c-8966040f11ae
+version: 1
+date: '2022-05-23'
+author: Michael Haag, Splunk
+type: Hunting
+datamodel:
+- Splunk_Audit
+description: 'The Splunk platform contains built-in search processing language (SPL) safeguards to warn you when you are about to unknowingly run a search that contains commands that might be a security risk. This warning appears when you click a link or type a URL that loads a search that contains risky commands. The warning does not appear when you create ad hoc searches. This warning alerts you to the possibility of unauthorized actions by a malicious user. Unauthorized actions include -
+  Copying or transferring data (data exfiltration), Deleting data and Overwriting data.
+  A possible scenario when this might occur is when a malicious person creates a search that includes commands that exfiltrate or damage data. The malicious person then sends an unsuspecting user a link to the search. The URL contains a query string (q) and a search identifier (sid), but the sid is not valid. The malicious person hopes the user will use the link and the search will run.
+  During analysis, pivot based on user name and filter any user or queries not needed. Adhoc queries found from dashboards, however in audittrail logs it does not show the dashboard name. The query defaults to adhoc and no Splunk system user activity.
```

Comment:
I would rephrase this a bit, "Adhoc queries found from dashboards" sounds confusing, are queries ran from dashboard seen as adhoc on the logs?